OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -101,11 +102,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </binding>
                         </bindingDesc>
                     </physDesc>
-                    
+
                     <history>
-                        <provenance> The provenance of the manuscript is unknown. Possibly, <ref target="#p2">unit 2</ref>  (a single bifolio) 
+                        <provenance> The provenance of the manuscript is unknown. Possibly, <ref target="#p2">unit 2</ref>  (a single bifolio)
```

Comment:
(not part of what you standardised, but I think we would capitalise "Unit" now, here and below)